### PR TITLE
Issue 20 remove favorite

### PR DESCRIPTION
**Pull request by hbellows**
#### Pivotal URL or Waffle Card number:
Continues work on issue #20

#### What does this PR do?
This PR adds a draft deleteFavorite function

#### Where should the reviewer start?
javascript/index.js

### CODE REVIEW

**Comment by hbellows** [comment]
@mikeyduece Yo - This is getting a little confusing.  I have the core functions set up, but I need to talk through how everything should work together with someone.  Can we talk tomorrow morning?

**Comment by mikeyduece** [comment]
Sure thing. Just text me when you're ready to start working. 👍

File: javascript/index.js [hunk 1]
```diff
@@ -37,28 +42,98 @@ const displayCurrentWeatherSummary = (response) => {
...
+     method: 'DELETE',
+     headers: { 'Content-Type': 'application/json' },
+     body: JSON.stringify(favoriteData)
+   })
+    .then(response => response.json())
+    .catch(error => console.error(error))
+    getFavorites();
+}
+
+// --------NOTES -> REMOVE ME BEFORE PUSH TO PRODUCTION--------
```
Comment (mikeyduece) [inline javascript/index.js]: Change your `NOTES` to `TODO: `

File: javascript/index.js [hunk 2]
```diff
@@ -37,28 +42,98 @@ const displayCurrentWeatherSummary = (response) => {
...
+    console.log(error)
+    });
+}
+
+const displayFavorites = (response) => {
+  $("#favorites").html('');
+  
+  let favorites = response.data
+  
+  favorites.forEach(function(location) {
```
Comment (mikeyduece) [inline javascript/index.js]: To keep your syntax the same I'd change this to 
```
favorites.forEach(location => {
  ...
})
```

Comment (hbellows) [reply javascript/index.js]: Awesome! This JS syntax gets tricky! Done and done!

**Pull request merged**